src/app.module.ts

```diff
@@ -23,6 +24,6 @@ import { Orders } from "./orders/domain/order.entity";
       synchronize: true
     })
   ],
-  controllers: [HealthController],
+  controllers: [CustomersController, ProductCategoryController, ProductController, OrdersController],
```

Comment:
**issue:** the controllers should be added to their own specific module and not in the root `app.module`.

**Reasoning:** we want to have each module have its own functionality as independent as possible for reusability

Reply:
Makes sense, noted!

Added the controllers associated with each feature in their own module like so:

![image](https://github.com/user-attachments/assets/99b5662a-7250-4c62-b0c5-dd601a791675)